pom.xml

```diff
@@ -7,13 +7,12 @@
 	<name>Calico Home Security</name>
 	<build>
 		<sourceDirectory>src/module_PI/Raspberry_PI</sourceDirectory>
-		<!--<testSourceDirectory>src/module_PI/Raspberry_PI/JUnitTests</testSourceDirectory>--> <!-- TODO: Adjust tests so they don't rely on an Arduino being plugged in (should be mocked) and running then uncomment this line so that all unit tests are run at build time -->
+		<testSourceDirectory>src/module_PI/Raspberry_PI/JUnitTests</testSourceDirectory> <!-- TODO: Adjust tests so they don't rely on an Arduino being plugged in 
```

Comment:
Were the tests changed so that the Arduino being plugged in isn't necessary? If not then these will fail on someone's machine who doesn't have one plugged in. The dependency on the Arduino should be mocked so it is no longer a dependency.